.github/workflows/demo_apps_build.yaml

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build example - Telink
+
+on:
+    push:
+
+concurrency:
+    group: ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name == 'pull_request' && github.event.number) || (github.event_name == 'workflow_dispatch' && github.run_number) || github.sha }}
+    cancel-in-progress: true
+
+env:
+    CHIP_NO_LOG_TIMESTAMPS: true
+
+jobs:
+    telink:
+        name: Telink
+        env:
+            BUILD_TYPE: telink
+            ZEPHYR_TOOLCHAIN_VARIANT: zephyr
+            ZEPHYR_SDK_INSTALL_DIR: /opt/telink/zephyr-sdk-0.15.2
```

Comment:
Make 0.15.2 as global define/variable, and use in all places bellow

Reply:
Done